cmd/access/node_builder/unstaked_access_node_builder.go

```diff
@@ -135,6 +135,11 @@ func (builder *UnstakedAccessNodeBuilder) Initialize() error {
 
 	builder.enqueueConnectWithStakedAN()
 
+	builder.EnqueueMetricsServerInit()
```

Comment:
This will mean metrics are also enabled for DPS. maybe we can just export `metricsEnabled` from `cmd.node_builder` and check it here?

Reply:
Good point. Hmm, ideally consensus follower should behave like an library not like a node.
So rather than enabling/disabling metrics we should expose them to be processed by calling host.
I just wonder if this PR is "good for now" ™️ and adding a ticket for further refactor?

Reply:
I'd say it's good for now if you don't think this change is going to cause problems with running DPS for mainnet16 (on v0.24)

Reply:
So once the Access Node / Observer have been refactored and separated into their own folders, they would probably both call the base [`Initialize()`](https://github.com/onflow/flow-go/blob/c622f81817fb148fd96a7576db23cbcfbf5b809b/cmd/scaffold.go#L1020) anyways.

At that point, if we want to keep supporting consensus follower, we will probably want to add new code for it instead of reusing the Observer code, and just initialize the components that are actually needed.

Reply:
https://github.com/onflow/flow-go/issues/2075